.github/workflows/production.yml

```diff
@@ -0,0 +1,19 @@
+name: Deploy to production
+
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on
+on:
+  push:
+    tags:
+      - v*
```

Comment:
I am concerned that this trigger is too permissive.   Would this allow anyone with repo push access to be able to deploy to production?  What we want is a trigger for tags which are pushed only to master.  We should be able to protect master branch sufficiently.

I could not figure out how to boolean AND workflow events (e.g. tag AND branch = master) but this SO describes one approach:  https://stackoverflow.com/questions/59874409/trigger-github-workflow-only-if-pushed-to-specific-branch-and-tag-exists

Something like (untested):
```
 - name: Exit if not on master branch
        if: github.ref != 'refs/heads/master')
        run: exit -1
```

Reply:
yeah, I presume we'd be protected in our repo setting but I much prefer the idea of an extra check here, I'll look into more and if I can't find something conventional from GH's docs, I'll use this exit approach.

Reply:
we'd have to make sure to get the property right, I just did a little test over in my sandbox and there might be a prop to go after that would be the one to check:  https://github.com/longrunningprocess/gha-sandbox/runs/4861490865?check_suite_focus=true#step:2:212

Reply:
I opened a ticket with GH just to make sure there isn't a built-in option but I'll go ahead and put in a fix for this today.

Reply:
I tested this approach out in my own sandbox so it should be good but we should plan on keeping an eye on it the next release we cut.  Fix in https://github.com/sillsdev/web-languageforge/pull/1284/commits/56308a9192616f08bd7e1f8887fd80c901c99bd2

Reply:
this looks good, thanks!